packages/esm-patient-banner-app/src/contact-details/contact-details.test.tsx

```diff
@@ -65,15 +62,45 @@ const mockPersonAttributes = [
   },
 ];
 
+const mockCohorts = [
+  {
+    uuid: 'fdc95682-e206-421b-9534-e2a4010cc05d',
+    name: 'List three',
+    startDate: '2023-04-19T23:26:27.000+0000',
+    endDate: null,
+  },
+  {
+    uuid: '1d48bec7-6aab-464c-ac16-687ba46e7812',
+    name: ' Test patient List-47',
+    startDate: '2023-04-24T23:28:49.000+0000',
+    endDate: null,
+  },
+  {
+    uuid: '6ce81b61-387d-43ab-86fb-606fa16d39dd',
+    name: ' Test patient List-41',
+    startDate: '2023-04-24T23:28:49.000+0000',
+    endDate: null,
+  },
+  {
+    uuid: '1361caf0-b3c3-4937-88e3-40074f7f3320',
+    name: 'Potential Patients',
+    startDate: '2023-06-07T15:40:00.000+0000',
+    endDate: null,
+  },
+];
 const mockOpenmrsFetch = openmrsFetch as jest.Mock;
 
 jest.mock('../hooks/usePatientAttributes.tsx', () => ({
   usePatientAttributes: jest.fn(),
   usePatientContactAttributes: jest.fn(),
 }));
 
+jest.mock('../hooks/usePatientListsForPatient.tsx', () => ({
+  usePatientListsForPatient: jest.fn(),
+}));
+
 describe('ContactDetails', () => {
-  it('renders an empty state view when relationships data is not available', async () => {
```

Comment:
I removed this suite since it was catered for in the empty view for all sections